src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -17,6 +17,7 @@
    [:CaService initialize-master-ssl! retrieve-ca-cert! retrieve-ca-crl! get-auth-handler]
    [:JRubyPuppetService]
    [:AuthorizationService wrap-with-authorization-check]
+   [:SchedulerService interspaced stop-job]
```

Comment:
I think if we're worried about leaking threads on shutdown from the job pool, tk-scheduler takes care of that for us by bringing it in as a service here: https://github.com/puppetlabs/trapperkeeper-scheduler/blob/master/src/puppetlabs/trapperkeeper/services/scheduler/scheduler_service.clj#L35-L40

So we probably don't need `stop-job`?